roles/ceph-mon/tasks/openstack_config.yml

```diff
@@ -50,6 +50,15 @@
     - openstack_config
     - item.0 != groups[mon_group_name] | last
 
+- name: chmod openstack key(s)
+  file:
+    path: "/etc/ceph/{{ cluster }}.{{ item.name }}.keyring"
+    mode: "{{ item.mode|default(omit) }}" # if mode not in list, uses mode from ps umask
+  with_items: "{{ openstack_keys }}"
+  when:
+    - openstack_config
+    - cephx
+    
```

Comment:
should this account for item.mode as well?

Reply:
Thanks, for pointing this out as I had to clean out a "keys.mode is defined" in create_users_keys.yml which I did in this latest revision. 

In this case, I'm relying on default(omit) [1] to handle the case where the key is not in the data structure. 

[1] http://docs.ansible.com/ansible/latest/playbooks_filters.html#omitting-parameters